packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -127,4 +128,54 @@ PouchDB.fetch = function (url, opts) {
   return fetch(url, opts);
 };
 
+function ActiveTasks() {
+  this.tasks = [];
+}
+
+ActiveTasks.prototype.list = function () {
+  return this.tasks;
+};
+
+ActiveTasks.prototype.add = function (task) {
+  const newTask = {
+    name: task.name,
+    total_items: task.total_items,
+    id: uuidv4().substr(0,8),
+    createdAt: new Date().toJSON()
+  };
+  this.tasks.push(newTask);
+  return newTask.id;
+};
+
+ActiveTasks.prototype.get = function (id) {
+  const index = this.tasks.findIndex(task => task.id === id);
+  return this.tasks[index];
+};
+
+ActiveTasks.prototype.remove = function (id) {
+  const index = this.tasks.findIndex(task => task.id === id);
+  if (index !== -1) {
+    this.tasks.splice(index, 1);
+  }
+  return this.tasks;
+};
+
+ActiveTasks.prototype.update = function (id, updatedTask) {
+  const index = this.tasks.findIndex(task => task.id === id);
+  if (index !== -1) {
+    const mergedTask = {
+      id: this.tasks[index].id,
+      name: this.tasks[index].name,
+      createdAt: this.tasks[index].createdAt,
+      total_items: updatedTask.total_items || this.tasks[index].total_items,
+      completed_items: updatedTask.completed_items || this.tasks[index].completed_items,
```

Comment:
why not have consistent camelCase or snake_case here?

Reply:
Yeah this makes no sense, good call. PouchDB generally uses 🐍 for options, so we should do that too.

Reply:
there's also the `createdAt` & `updatedAt` convention with Couch (in camelCase), but we're not accessing a database here... i'm not sure! searching for `updated_at` in the pouch repo doesn't yield a single result.

Reply:
the convention is just about collecting those attributes, not necessarily its camel-case-ness. so `created_at` etc. is fine to match PouchDB conventions. #teamsnake